addon/components/file-dropzone.ts

```diff
@@ -0,0 +1,354 @@
+import Component from '@glimmer/component';
+import { inject as service } from '@ember/service';
+import { getOwner } from '@ember/application';
+import DataTransferWrapper, {
+  FileUploadDragEvent,
+} from '../system/data-transfer-wrapper';
+import { action } from '@ember/object';
+import { tracked } from '@glimmer/tracking';
+import Queue from '../queue';
+import UploadFile, { FileSource } from 'ember-file-upload/upload-file';
+import FileQueueService, { DEFAULT_QUEUE } from '../services/file-queue';
+import { modifier } from 'ember-modifier';
+
+interface FileDropzoneArgs {
+  queue?: Queue;
+
+  /**
+   * Whether users can upload content from websites by dragging images from
+   * another webpage and dropping it into your app. The default is `false`
+   * to prevent cross-site scripting issues.
+   * */
+  allowUploadsFromWebsites?: boolean;
```

Comment:
Is that argument already present? It starts with a verb, so it indicates it is callable, ie `allowUploadsFromWebsites()` - it is a _question_ rather than a _fact_. As _fact_ I'd call it `uploadsFromWebsitesAllowed`.

Reply:
This is an existing argument. I'd prefer not to change it for now but can revisit in future.